src/seccomp/policy/DefaultPolicy.cc

```diff
@@ -49,29 +49,30 @@ void DefaultPolicy::addExecutionControlRules(bool allowFork) {
 
     rules_.emplace_back(SeccompRule(
             "execve",
-            action::ActionTrace([executed = false](auto& tracee) mutable {
-                if (executed)
+            action::ActionTrace([executed = false](auto& /* tracee */) mutable {
+                if (executed) {
                     return tracer::TraceAction::KILL;
+                }
                 executed = true;
                 return tracer::TraceAction::CONTINUE;
             })));
 
     for (const auto& syscall: {"kill", "tkill"}) {
-        rules_.emplace_back(
-                SeccompRule(syscall, action::ActionTrace([](auto& tracee) {
-                                if (isSignalValid(tracee.getSyscallArgument(1)))
-                                    return tracer::TraceAction::CONTINUE;
-                                else
-                                    return tracer::TraceAction::KILL;
-                            })));
+        rules_.emplace_back(SeccompRule(
+                syscall, action::ActionTrace([](auto& tracee) {
```

Comment:
IMO, `syscall,` w osobnej linii

Reply:
Da się pewnie do dobrać odpowiednimi ustawieniami .clang-format, ale... Ja akceptuje to jako cene za spójne (i automatyczne) formatowanie kodu.